### PR TITLE
removed some leftover references to Greeting

### PR DESCRIPTION
**Pull request by cblanto7**
deleted the remaining greeting-related stuff: the template and removed the link to greeting from our index page.
Added one single test in the StudentControllerTest.

### CODE REVIEW

File: src/test/java/unbanner/StudentControllerTest.java [hunk 1]
```diff
@@ -2,12 +2,14 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-
 import org.junit.Test;
+import org.junit.Assert.*;
```
Comment (zeonin) [inline src/test/java/unbanner/StudentControllerTest.java]: Can you unglob this? Style check doesn't like '.*'

Also, this should be above org.junit.Test

Comment (nbarnes7) [reply src/test/java/unbanner/StudentControllerTest.java]: import static org.junit.Assert.assertEquals; 
is this okay with Style Check?

Comment (zeonin) [reply src/test/java/unbanner/StudentControllerTest.java]: Yep, that should work. Style Checker just wants imports to be explicit and in alphabetical order.

**Comment by cblanto7** [comment]
made the changes requested

**Pull request merged**